core/lib/constants/src/system_logs.rs

```diff
@@ -1,8 +1,8 @@
 /// The key of the system log with value of the L2->L1 logs tree root hash
 pub const L2_TO_L1_LOGS_TREE_ROOT_KEY: u32 = 0;
 
-/// The key of the system log with value of the state diff hash
-pub const STATE_DIFF_HASH_KEY: u32 = 2;
+/// The key of the system log with value of the state diff hash for pre-gateway protocol versions
```

Comment:
what is in that slot AFTER gateway? 
(did we move slot numbers or do we simply put empty data there)

Reply:
we've moved slots around, IIUC some were removed and some new were added. After gateway `PREV_BATCH_HASH_KEY` has key = 2